### PR TITLE
Holded state

### PR DESCRIPTION
**Pull request by d-nakamichi**
It adds HOLDED state for holding ROS robot control.
It also redefines state transitions.

### CODE REVIEW

File: khi_robot_control/src/khi_robot_krnx_driver.cpp [hunk 1]
```diff
@@ -412,16 +424,30 @@ bool KhiRobotKrnxDriver::activate( const int cont_no, JointData *joint )
...
+    int state;
+
+    if ( !contLimitCheck( cont_no, KRNX_MAX_CONTROLLER ) ) { return false; }
+
+    state = getState( cont_no );
+    if ( state == ACTIVE )
+    {
+        setState( cont_no, HOLDED );
+    }
+}
```
Comment (matsui-hiro) [inline khi_robot_control/src/khi_robot_krnx_driver.cpp]: `return true` is needed.

File: khi_robot_control/src/khi_robot_krnx_driver.cpp [hunk 2]
```diff
@@ -829,30 +897,40 @@ bool KhiRobotKrnxDriver::makeRtcParam( const int cont_no, const std::string name
...
-            snprintf( cmd_buf, sizeof(cmd_buf), "HERE/N %d: #rtchome%d", ano+1, ano+1 );
-            return_code = krnx_ExecMon( cont_no, cmd_buf, msg_buf, sizeof(msg_buf), &error_code );
+    return_code = krnx_Load( cont_no, file_path );
+    unlink( file_path );
+    if ( !retKrnxRes( cont_no, "krnx_Load", return_code ) ) { return false; }
+
+    return true;
+}
+
+bool KhiRobotKrnxDriver::syncRtcPos( const int cont_no, JointData *joint )
```
Comment (matsui-hiro) [inline khi_robot_control/src/khi_robot_krnx_driver.cpp]: `JointData *joint` is not used in this func.

Comment (d-nakamichi) [reply khi_robot_control/src/khi_robot_krnx_driver.cpp]: Yes, I removed argument `joint`.

File: khi_robot_control/src/khi_robot_krnx_driver.cpp (hunk 1 above)
Comment (d-nakamichi) [reply khi_robot_control/src/khi_robot_krnx_driver.cpp]: Yes, I added `return true`.

**Comment by matsui-hiro** [comment]
I confirmed changes.

**Pull request merged**